### ISSUE TITLE
avoid-inline-spacing should consider the values

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3467]
Axe-core should not just look at whether or not `!important` is used, but also whether or not the value is below the threshold WCAG sets for it. For example the following should not fail the `avoid-inline-spacing` rule, because 0.15em is sufficient to satisfy WCAG:

```html
<p style="letter-spacing: 0.15em !important">
  The toy brought back fond memories of being lost in the rain forest.
</p>
```

There are quite a few ACT examples axe-core is getting wrong because it's not considering the value, or how it's set:

- [Letter spacing in `style` attributes is not `!important`](https://www.w3.org/WAI/standards-guidelines/act/rules/24afc2/)
   - [Passed example 2](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/9e9382901f59c7dd476717a55bf5c5a37ed76bbc.html)
   - [Passed example 3](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/43f8fe88b8e7365db7aa251b263b5d00c7a47ae9.html)
   - [Passed example 4](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/787f24a573fa422e24ab72312f7306253bb83a4f.html)
   - [Passed example 5](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/f000a9c495f11a4a11a4314871b91f4173e4589a.html)
   - [Passed example 7](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/6aa2034507dc16e6ae0d16f1b6f2a14d3dfadc18.html)
   - [Passed example 8](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/64b25817b3d3909ab7f4acaee061875ebac1cee3.html)
   - [Inapplicable example 8](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/88d6ea5706ed8ae188caa166879c381e64e5077a.html)

- [Word spacing in `style` attributes is not `!important`](https://www.w3.org/WAI/standards-guidelines/act/rules/9e45ec/)
   - [Passed example 2](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/45e5a588c3e8977fa0e83074d7f7c89738e8ec42.html)
   - [Passed example 3](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/2a2a14cc9bcb3fa7983e22f160ce9eeb6b832a8c.html)
   - [Passed example 4](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/6d5dde208ef91b6afceca022c7a2a12b99f042b7.html)
   - [Passed example 5](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/2d9b8cf0906f0e05e4d487c9682db7a7e022fab0.html)
   - [Passed example 7](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/fa119442cf663c73bf332488f3965b427b024009.html)
   - [Passed example 8](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/edaf06132468eccf5fd90551151252a364b44b7b.html)
   - [Inapplicable example 3](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/24afc2/a8f0c6682763e4ca7db824dc145a23067a3eb889.html)

**Comment by padmavemulapati** [comment]
Validated with the latest develop branch code base, with the script:
```
<!-- Letter spacing in style attributes is not !important -->
  <p style="letter-spacing: 0.15em !important">
    The toy brought back fond memories of being lost in the rain forest.
  </p>

  <title>Passed Example 2</title>
</head>
<body>
	<p style="letter-spacing: 0.15em !important">
		The toy brought back fond memories of being lost in the rain forest.
	</p>
</body>
<title>Passed Example 3</title>
<style>
  p {
    font-size: 25px;
  }
</style>

<p style="letter-spacing: 3px !important">
  The toy brought back fond memories of being lost in the rain forest.
</p>
<p style="letter-spacing: 0.1em !important; letter-spacing: 0.15em !important">
  The toy brought back fond memories of being lost in the rain forest.
</p>
<p style="letter-spacing: 0.15em !important; letter-spacing: 0.1em">
  The toy brought back fond memories of being lost in the rain forest.
</p>
<p style="letter-spacing: 0.1em">
  <span style="letter-spacing: inherit !important;">
    The toy brought back fond memories of being lost in the rain forest.
  </span>
</p>
<title>Inapplicanle ex</title>
<p style="position: absolute; top: -999em; letter-spacing: 0.1em !important;">
  The toy brought back fond memories of being lost in the rain forest.
</p>

```
<img width="1304" alt="image" src="https://user-images.githubusercontent.com/47409579/187223203-82db8b75-b503-4803-8227-6b5f401836ad.png">
